pkgs/development/node-packages/node-packages-v10.nix

```diff
@@ -1872,7 +1872,7 @@ let
       packageName = "strip-ansi";
       version = "3.0.1";
       src = fetchurl {
-        url = "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz";
+        url = "http://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz";
```

Comment:
Oh actually, this is weird, why did it switch to http? Can you maybe run ./generate.sh again with the latest master version?

Reply:
Not coming back :'( It did get two updates though, so I did a push -f